docs/api/web-contents.md

```diff
@@ -1519,13 +1519,19 @@ console.log(requestId)
 * `opts` Object (optional)
   * `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
   * `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
+  * `outputSize` [Size](structures/size.md) (optional) - The desired width and height in pixels (positive numbers) of the returned image.
 
 Returns `Promise<NativeImage>` - Resolves with a [NativeImage](native-image.md)
 
 Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page.
 The page is considered visible when its browser window is hidden and the capturer count is non-zero.
 If you would like the page to stay hidden, you should ensure that `stayHidden` is set to true.
 
+The size of returned [NativeImage](native-image.md) will be `outputSize` if it was provided.
+Otherwise, the size would be large enought to fit all of the available pixels in the current system.
```

Comment:
We should validate this with a test! Ex. validate that if the default size would be `{ 200, 200 }` and i set outputSize to `{ 1500, 150 }` the resultant image should have a size of `{ 1500, 150 }`.

Reply:
Good suggestion!
I've elaborated the test into three scenarios: 
1. requested size is smaller than captured area
2. one dimension of requested size is bigger than captured area
3. both dimensions are bigger than captured area